packages/top-app-bar/src/mwc-top-app-bar-base-base.ts

```diff
@@ -42,6 +42,8 @@ export abstract class TopAppBarBaseBase extends BaseElement {
 
   private _scrollTarget!: HTMLElement|Window;
 
+  @property({type: Boolean, reflect: true}) centerTitle = false;
```

Comment:
Does this need to be reflected? What are the reasons for?

Reply:
It was `reflect: true` previously, but I don't think that actually matters. I'll remove that.